### PR TITLE
Updated docs to clarify a few CX questions.

### PR DESCRIPTION
**Pull request by daravep**

### CODE REVIEW

File: docs/2.8.0/docs/canton/usermanual/upgrading.rst [hunk 1]
```diff
@@ -179,6 +181,9 @@ Finally, you can ping the participant to see if the system is operational
 .. snippet:: migrating_participant
     .. success:: participant.health.ping(participant)
 
+The ping command involves a create, subsequent exercise and archive of two contracts
+between the admin parties, which means that the full ledger functionality is tested end-to-end.
```
Comment (marciw-da) [inline docs/2.8.0/docs/canton/usermanual/upgrading.rst:185]:
```suggestion
The ping command creates two contracts between the admin parties, then exercises and archives them -- providing an end-to-end test of ledger functionality.
```

alt: "serving as an end-to-end test"

File: docs/2.8.0/docs/canton/usermanual/upgrading.rst [hunk 2]
```diff
@@ -378,8 +383,9 @@ must be unique.
 Therefore, the protocol upgrade process boils down to:
 
 - Deploy a new domain next to the old domain. Ensure that the new domain is using the desired protocol version.
-  Ensure that you are using different databases (or at least different schemas in the same database), channel names, smart contract addresses etc.
-  It must be a completely separate domain (albeit you can reuse your DLT backend as long
+  Ensure that you are using different databases (or at least different schemas in the same database)
+  for all the domain nodes (mediator, sequencer node and topology manager), channel names, smart contract addresses etc.
+  It must be a completely separate, new domain (albeit you can reuse your DLT backend as long
```
Comment (marciw-da) [inline docs/2.8.0/docs/canton/usermanual/upgrading.rst:388]:
```suggestion
  Also make sure to use different databases (or at least different schemas in the same database)
  for the domain nodes (mediator, sequencer node, and topology manager), channel names, smart contract addresses, etc.
  The new domain must be completely separate, but you can reuse your DLT backend as long
```

File: docs/2.8.0/docs/canton/usermanual/upgrading.rst [hunk 3]
```diff
@@ -378,8 +383,9 @@ must be unique.
...
   as you use different sequencer contract addresses or Fabric channels).
```
Comment (marciw-da) [inline docs/2.8.0/docs/canton/usermanual/upgrading.rst:389]:
```suggestion
  as you use different sequencer contract addresses or Fabric channels.
```

(apply this change only if you apply the preceding change that removes the open paren)

File: docs/2.8.0/docs/canton/usermanual/upgrading.rst [hunk 4]
```diff
@@ -133,13 +133,15 @@ observe the following Flyway error:
...
-database schema version and how many migrations need to be applied.
+database schema version and how many migrations need to be applied. This check runs on every
+startup of the node, so if the node starts, the migrations were successful.
 
 We can now force the migration to a new schema using:
 
 .. snippet:: migrating_participant
     .. success:: participant.db.migrate()
 
+You can also configure the :ref:`migrations to be applied automatically <migrate_and_start_mode>`.
```
Comment (marciw-da) [inline docs/2.8.0/docs/canton/usermanual/upgrading.rst:144]:
```suggestion
You can also :ref:`configure the migrations to be applied automatically <migrate_and_start_mode>`.
```

**Review by marciw-da — APPROVED** [review]
Suggestions for clarity and readability.

Approving so you can apply the suggestions and merge -- but if you decide not to apply the suggestions, would like to discuss or re-review.

**Comment by daravep** [comment]
@marciw-da looks like @hrischuk-da applied your suggestions, as they are "outdated" now. i'll merge. 
btw, i don't have strong feelings about my written text. given that i'm not a native speaker, i don't assume at all that my wording is great. so generally, please feel free to edit my texts at your own discretion.

**Pull request merged**